### ISSUE TITLE
check accessability of project/module toggle in dashboard

### ISSUE THREAD

**Issue by Rineee** [liqd/a4-meinberlin#3501]
**URL:** 
**user:** 
**expected behaviour:** 
**behaviour:** 
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?

**Comment by phillimorland** [comment]
You can currently toggle the project/module dropdown to open and close, however even if it's closed, it will still tab through all the items, this is because we use non standard element for the toggleing (checkbox) as it was decided we needed to remember which dropdowns were open so we set a cookie to remember if it was checked or not. I think only way to change this is to either re-write the cookie to determine and remember if a dropdown is expanded or not, there were certainly issues trying to do this when we built it but maybe now it's easier?! or we use js to remove the menu items from the tabbing order when checkbox is not checked which I think should be relativly easy, might just have to faff a bit. Also when we do this we should add a aria labelled by to the checkbox.
@CarolingerSeilchenspringer it would be your call here?

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland @Rineee isn't that the same issue as this one you closed? https://github.com/liqd/a4-meinberlin/issues/3480

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer kind of, my understanding of that issue was a keyboard user could not open the menus, this was fixed with a pull request and so issue was auto closed. The tabbing through closed menus I wasn't aware of till reading this issue and checking, I then commented, i'm not sure what triggered @Rineee to trigger this issue?

**Comment by Rineee** [comment]
Yes, I meant exactly what you wrote above @phillimorland, that users still tab through the menu even if its closed. I am sorry for not being more specific about it before.. So I guess removing the menu items from tabbing (with js) would be good, what do you think @CarolingerSeilchenspringer?

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland @Rineee I guess I also wasn't specific enough. In my issue I meant what rine wrote in her issue (have to tab through menu even if closed) which is of course connected to not be able to even open/close the navigation. So your solution is that we 1) make it possible to open/close navigation through tabbing (which should be there but I don't know how to do it, @phillimorland can you tell me?) and 2) if a navigation is closed and I don't want to open it I can tab right to the next button that can actually be seen. Correct? Sounds good to me!

**Comment by phillimorland** [comment]
you open with space bar, this is clear if using a screen reader, for non screen reader tab users, we will have to hope it isn't too unexpected as not sure what other option we have

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland thanks. I didn't get that because there is no focus on the arrow.  The focus is on "alle Beteiligungsprojekte", then nowhere (should be the arrow) and then on the top of the left navigation where project name is shown. Can we make it more clear? ok, but this doesn't belong into this issue, so I will copy this comment to the other one. :-)

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer not with how we built this, the requirment that the menus have to stay open after a page reload means we had to hack something together as it's non standard bevaviour for collasable elements (dropdowns, accordions), we should have maybe been clearer about that when building it. By haveing to make a frankenstein style element to fulfill that requirment we paid for it with making it less accessible because it's non standard, whenever we implement non standard behaviour, there will be an accessibility cost unfortunatly. I can see if I can manipulate the focus ring but I won't promise anything.
